TSEconomy/Logging/TransactionLogging.cs

```diff
@@ -35,7 +35,9 @@ public static void Log(Transaction trans)
 
             using (StreamWriter writer = File.AppendText(sessionLogFile))
             {
-                writer.WriteLine($"[{trans.Timestamp}] : {trans.TransactionDetails}");
+                string flag = (trans.Flags & TransactionProperties.set) == TransactionProperties.set ? "Set" : "Add";
```

Comment:
what is this doing?

Reply:
> specifying whether or not the transaction is a Set type or an Add type in the log message, since the plugin is gonna get used as an API, we cant be sure whether or not other plugins will specify this in their details

ah okay, i see thanks